lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -597,14 +619,16 @@ class EmitterBase {
 } // end anonymous namespace
 
 void EmitterBase::emitTextWithSubstitutions(
-    StringRef string, Operation *op,
-    std::function<void(Value)> operandEmitter) {
+    StringRef string, Operation *op, std::function<void(Value)> operandEmitter,
+    llvm::Optional<SmallVector<Operation *, 8>> symOps,
+    llvm::Optional<ModuleNameManager> names) {
```

Comment:
Won't this copy the ModuleNameManager?  I'd recommend something like a possibly-null `ModuleNameManager*`.

Reply:
Yes, it was copying it. Fixed this and removed the Optional stuff.